recipe/meta.yaml

```diff
@@ -1,72 +1,96 @@
-{% set version = "2.4.12" %}
-{% set filename = "opencv2-%s.zip" % version %}
+{% set version = "3.1.0" %}
+
 
 package:
   name: opencv
   version: {{ version }}
 
 source:
-  fn: {{ filename }}
-  url: https://github.com/Itseez/opencv/archive/{{ version }}.zip
-  sha1: 2225768a48dbf99426bf77451a433ed432f54d8e
+  fn: opencv-{{ version }}.tar.gz
+  url: https://github.com/Itseez/opencv/archive/{{ version }}.tar.gz
+  sha256: f00b3c4f42acda07d89031a2ebb5ebe390764a133502c03a511f67b78bbd4fbf
 
   patches:
-    - osx_rpath.patch  # [osx]
+    - cap_mpjpeg_decoder.patch      # [win]
```

Comment:
Could we please have a comment above this patch with a link to commit/PR that fixed it?